test/cpp/qps/client.h

```diff
@@ -344,6 +346,7 @@ class Client {
   bool closed_loop_;
   gpr_atm thread_pool_done_;
   double median_latency_collection_interval_seconds_;  // In seconds
+  bool all_channels_connected_ = true;
```

Comment:
nit: seems a little odd to initialize `true`. Can we initialize `false` and set true and the end of `WaitForChannelsToConnect` ?

Reply:
Done.